src/theory/theory_inference_manager.cpp

```diff
@@ -34,11 +36,18 @@ TheoryInferenceManager::TheoryInferenceManager(Theory& t,
       d_lemmasSent(t.getUserContext()),
       d_numConflicts(0),
       d_numCurrentLemmas(0),
-      d_numCurrentFacts(0)
+      d_numCurrentFacts(0),
+      d_inferenceStats(name + "::inferenceIds")
```

Comment:
If we split this into 3, maybe this would be `::inferencesFact`, `::inferencesLemma` and `::inferencesConflict`.